src/rstar.jl

```diff
@@ -109,16 +108,25 @@ is returned (algorithm 2).
 # Examples
 
 ```jldoctest rstar; setup = :(using Random; Random.seed!(101))
-julia> using MLJBase, MLJXGBoostInterface, Statistics
+julia> using MLJBase, MLJIteration, EvoTrees, Statistics
 
 julia> samples = fill(4.0, 100, 3, 2);
 ```
 
-One can compute the distribution of the ``R^*`` statistic (algorithm 2) with the
+One can compute the distribution of the ``R^*`` statistic (algorithm 2) with a
 probabilistic classifier.
 
 ```jldoctest rstar
-julia> distribution = rstar(XGBoostClassifier(), samples);
+julia> model = IteratedModel(;
```

Comment:
It's too bad that this setup is so much more verbose than just calling `XGBoostClassifier()`.

Reply:
Well, we can just use `EvoTrees(; nrounds=100, eta=x)` (don't remember the default in `XGBoostClassifier`) and would get the same setting since `XGBoostClassifier` just uses `nrounds = 100` by default without any tuning of this hyperparameter. Based on the comments above I thought it would be good though to highlight how it can be set/estimated in a better way. Maybe should add a comment though and show `EvoTrees(; nrounds=100)` as well.

Reply:
Yeah for a usage example in the docstring I slightly prefer the simpler approach. But I agree that it is good to document the more robust approach somewhere.